src/Symfony/Component/HttpFoundation/StreamedResponse.php

```diff
@@ -56,8 +56,12 @@ public function setCallback(callable $callback): static
         return $this;
     }
 
-    public function getCallback(): \Closure
+    public function getCallback(): ?\Closure
     {
+        if (!isset($this->callback)) {
```

Comment:
I would use `null === $this->callback` for consistency with our other null checks

Reply:
isset() is what we use on 7.0 because the property is not declared as nullable

Reply:
but this whole ticket is about fixing the fact that the callback can be null. So to me, the 7.0 type should also be changed as part of this fix.

Reply:
btw, as the property is protected and not private, this is a BC break to turn it into a non-nullable potentially-uninitialized property as the way to represent an existing state.